documentation/docs/liquidity-mining.md

```diff
@@ -134,4 +134,4 @@ Partnering with Harmony, we launched a successful pilot program rewarding commun
 
 [Zilliqa (ZIL)](https://zilliqa.com/) is a base-layer blockchain protocol centered around the idea of 'Sharding' and designed to enhance the scalability of cryptocurrency networks such as Ethereum. According to its white paper, transactions speed could scale to approximately 1,000x that of Ethereum's network. Zilliqa’s high throughput means that developers can focus on fleshing out their ideas rather than worrying about network congestion.
 
-[Whitepaper](https://docs.zilliqa.com/hummingbot.pdf) | [Twitter](https://twitter.com/zilliqa) | [Telegram](https://t.me/zilliqachat) | [Github](https://github.com/Zilliqa) | [Explorer](https://viewblock.io/zilliqa) | [CoinMarketCap](https://coinmarketcap.com/currencies/zilliqa) | [CoinGecko](https://www.coingecko.com/en/coins/zilliqa)
\ No newline at end of file
+[Whitepaper](https://https://docs.zilliqa.com/whitepaper.pdf) | [Twitter](https://twitter.com/zilliqa) | [Telegram](https://t.me/zilliqachat) | [Github](https://github.com/Zilliqa) | [Explorer](https://viewblock.io/zilliqa) | [CoinMarketCap](https://coinmarketcap.com/currencies/zilliqa) | [CoinGecko](https://www.coingecko.com/en/coins/zilliqa)
```

Comment:
@rennel-tabing i think you input `https://` twice here.

Reply:
Thanks for catching that up.  @PtrckM